src/module/party/party-xp.js

```diff
@@ -46,7 +46,8 @@ export default class OsePartyXP extends FormApplication {
   }
 
   // eslint-disable-next-line no-underscore-dangle
-  _onDrop(event) { // eslint-disable-line class-methods-use-this
+  _onDrop(event) {
+    // eslint-disable-line class-methods-use-this
```

Comment:
eslint-disable-line dropped down onto its own line? Prettier wyd

Reply:
Hahaha heck. I forgot you could combine disables. This should be good now.